spec/translated_properly_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+RSpec.describe Card, :type => :model do
+  describe('original_text not equal translated_text') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.